src/oscar/locale/en_US/LC_MESSAGES/django.po

```diff
@@ -5347,10 +5342,6 @@ msgstr ""
 msgid "Go to homepage"
 msgstr ""
 
-#: templates/oscar/flatpages/default.html:11
-msgid "Edit this page"
-msgstr ""
-
 #: templates/oscar/login_forbidden.html:5
 msgid "This page is only available to users who aren't logged in."
 msgstr ""
```

Comment:
Ich glaube die POs müssen nicht bei jeder Änderung aktualisiert werden.

Reply:
Ich aktualisiere die PO's, um alle Referenzen zu tilgen.
Da ich sie über `make messages` erzeuge ist es jetzt nicht so der große Aufwand, dass mit in meinen Worflow des Löschen mit auf zu nehmen.

Reply:
Richtig. Mein Punkt ist: du könntest das ja auch nicht tun.

Reply:
Dadurch gibt es nur eine Menge diff-noise, und beim rebase muss man sich drum kümmern.

Reply:
Ja, wegen dem diff noise haste Recht.